doc/server_API.md

```diff
@@ -264,6 +250,36 @@ On success:
 | 404 (Not Found) | Whitelist with such id or module does not exists |
 
 
+##add_plate_to_all_whitelists
```

Comment:
I think it would be helpful to add some descriptions to all of those calls.

For example, for somebody that only just started reading this documentation, it is not clear that those calls would be per user - i.e. that call `POST /add_plate_to_whitelist/` could mean adding one plate to all whitelists, which should not be permitted.

I think that little description, for example: "This route adds a plate to all whitelists bound to current user" would suffice and would greatly improve readability.